quickcheck/aefate_eqc.erl

```diff
@@ -0,0 +1,107 @@
+%%% @author Thomas Arts
```

Comment:
I see moved from https://github.com/Quviq/epoch-eqc/blob/e2b1542f77c1511c5fc400861877b0c26bd27193/aefate_eqc/aefate_eqc.erl (I did not check exact file content).
Ok.

Reply:
The model is updated. This one works, contra the old one. Better to keep them together

Reply:
See https://github.com/Quviq/epoch-eqc/pull/7